apps/ws/src/readiness/readiness.module.ts

```diff
@@ -0,0 +1,47 @@
+import { Module, Provider } from '@nestjs/common';
+import {
+  DalServiceHealthIndicator,
+  IHealthIndicator,
+  ReadinessService,
+  WebSocketsQueueService,
+  WebSocketsQueueServiceHealthIndicator,
+} from '@novu/application-generic';
+import { WSServerHealthIndicator } from '../socket/services';
+import { SharedModule } from '../shared/shared.module';
+import { SocketModule } from '../socket/socket.module';
+
+export const indicatorList = {
+  provide: 'INDICATOR_LIST',
+  useFactory: (
+    wsServerHealthIndicator: WSServerHealthIndicator,
+    webSocketsQueueServiceHealthIndicator: WebSocketsQueueServiceHealthIndicator,
+    dalServiceHealthIndicator: DalServiceHealthIndicator
```

Comment:
Unlike the worker implementation, here i added all the service indicators and not only the queue. would love to hear what you think about it.